shop/src/main/java/ro/msg/learning/shop/configuration/mail/EmailConfiguration.java

```diff
@@ -0,0 +1,57 @@
+package ro.msg.learning.shop.configuration.mail;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+@PropertySource(value = {"classpath:application.properties"})
+public class EmailConfiguration {
+    @Value("${spring.mail.host}")
+    private String mailServerHost;
+
```

Comment:
Try to be consistent with the empty lines, either add one between each entry or tie all of them together